checks/checks.yaml

```diff
@@ -144,6 +162,111 @@ checks:
+      (codeql) and sonarcloud in the recent (~30) merged PRs. The
+      check also looks for the use of "github/codeql-action" in a github workflow.
+    remediation:
+      - >-
+        Run CodeQL checks in your CI/CD by following the instructions
+        [here](https://github.com/github/codeql-action#usage).
+  Packaging:
+    risk: Medium
+    description:
+      This check tries to determine if the project is published as a package that other developers can install/download.
+
+      Packaging your project is important for users to receive updates and security patches automatically.
+      A low score is considered `Medium` risk.
+
+      The check currently looks for [GitHub packaging workflows]( https://docs.github.com/en/packages/learn-github-packages/publishing-a-package)
+      and language-specific GitHub Actions that upload the package to a correpdonsing hub, e.g., [Npm](https://www.npmjs.com/).
+      There is a plan to add better support to query package manager hubs directly in the future, e.g., for [Npm](https://www.npmjs.com/), [PyPi](https://pypi.org/).
+    remediation:
+      - Publish your project as a [downloadable package](https://docs.github.com/en/packages/learn-github-packages/publishing-a-package).
+      - Use a GitHub action to release your package to language-specific hubs.
+  Signed-Releases:
+    risk: High
+    description: >-
+      This check tries to determine if a project cryptographically signs release
+      artifacts. 
+      
+      Signed releases attest to the provenance of the artifact.
+      A low score is considered 'High' risk.
+
+      It works by looking for filenames: *.minisign
+      (https://github.com/jedisct1/minisign), *.asc (pgp), *.sign. for the last
+      5 GitHub releases. The check does not verify the signatures.
+    remediation:
+      - Publish the release.
+      - Generate a signing key.
+      - Download the release as an archive locally.
+      - Sign the release archive with this key (should output a signature file).
+      - Attach the signature file next to the release archive.
+      - >-
+        For GitHub, check out the steps
+        [here](https://wiki.debian.org/Creating%20signed%20GitHub%20releases).
+  Signed-Tags:
+    risk: Medium
+    description: >-
+      This check looks for cryptographically signed tags in the last 5 tags.
+      
+      Signed tags attest the author of a commit. A low score is considered 'Medium' risk.
+      
+      The check does not verify the signature itself and currently relies on github's verification.
+    remediation:
+      - Generate a new signing key.
+      - Add your key to your source hosting provider.
+      - Configure your key and email in git.
+      - Publish the tag and then sign it with this key.
+      - >-
+        For GitHub, check out the steps
+        [here](https://docs.github.com/en/github/authenticating-to-github/signing-tags#further-reading).
+  Pull-Requests:
```

Comment:
I think we should remove this check. See https://github.com/ossf/scorecard/issues/731